lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -440,4 +440,14 @@ defmodule LivebookWeb.SessionLive.CellComponent do
   end
 
   defp evaluated_label(_time_ms), do: nil
+
+  defp html_input_type(:password), do: "password"
+
+  defp html_input_type(:number), do: "number"
+
+  defp html_input_type(:url), do: "url"
+
+  defp html_input_type(:color), do: "color"
+
```

Comment:
```suggestion

  defp html_input_type(:number), do: "number"
  defp html_input_type(:url), do: "url"
  defp html_input_type(:color), do: "color"
```

Reply:
Removed all these newlines. Is the general style rule here that one-line implementations of the same function don't have newlines between them?

Reply:
Yup, that's a widely applied rule of thumb :)